CONVENTIONS.md

```diff
@@ -199,6 +199,12 @@ To avoid disruption and mass pod-death, it is important to
   * Components that support workloads directly must not disrupt end-user workloads during upgrade or reconfiguration
     * E.g. the upgrade of a network plugin must serve pod traffic without disruption (although tiny increases in latency are allowed)
     * All components that currently disrupt end-user workloads must prioritize addressing those issues, and new components may not be introduced that add disruption
+* All dameonsets in OpenShift should be at `maxUnavailable: 10%`.
```

Comment:
```suggestion
* All dameonsets of OpenShift components should use the maxUnavailable rollout strategy, allowing up to 10% of instances to be taken down at once.
```

Reply:
```suggestion
* All rollingUpdate dameonsets in OpenShift should utilize `maxUnavailable: 10%` to ensure a timely rollout on large clusters.
```
The phrase "rollout strategy" seems a bit too close to the spec field names in my opinion. here's what it actually looks like
```
spec:
  updateStrategy:
    type: RollingUpdate
    rollingUpdate:
      maxUnavailable: 10%
```

Reply:
Would it make sense to put that YAML snippet right into the text?